internal/services/springcloud/spring_cloud_accelerator_resource.go

```diff
@@ -27,9 +28,17 @@ type SpringCloudAcceleratorModel struct {
 
 type SpringCloudAcceleratorResource struct{}
 
+func (s SpringCloudAcceleratorResource) DeprecationMessage() string {
+	if !features.FivePointOh() {
+		return ""
+	}
+	return "Azure Spring Apps is now Deprecated and will be Retired on 2028-05-31 - as such the `azurerm_spring_cloud_accelerator` resource is deprecated and will be removed in v6.0 of the AzureRM Provider. See https://aka.ms/asaretirement for more information."
```

Comment:
We have a function for this. Can you please update all the other deprecation messages to use this instead of the `FivePointOh` flag directly and also update the deprecation message with the changes below
```suggestion
	if features.DeprecatedInFivePointOh() {
		return "Azure Spring Apps is deprecated and will be retired on 2028-05-31, as a result the `azurerm_spring_cloud_accelerator` resource is deprecated and will be removed in v6.0 of the AzureRM Provider. See https://aka.ms/asaretirement for more information."
	}
```

Reply:
Got it!